### PR TITLE
Display multiple module dependency errors when saving default project settings

### PR DESCRIPTION
**Pull request by dombesz**
https://community.openproject.org/work_packages/52660

Follow up for: #14732

### CODE REVIEW

File: app/controllers/admin/settings/projects_settings_controller.rb [hunk 1]
```diff
@@ -45,14 +45,17 @@ def validate_enabled_modules
 
       module_missing_deps = OpenProject::AccessControl
         .modules
-        .detect { |m| m[:dependencies] && enabled_modules.include?(m[:name]) && (m[:dependencies] & enabled_modules) != m[:dependencies] }
+        .select do|m| m[:dependencies] && enabled_modules.include?(m[:name]) && (m[:dependencies] & enabled_modules) != m[:dependencies] }
```
Comment (aaron-contreras) [inline app/controllers/admin/settings/projects_settings_controller.rb]:
```suggestion
        .select { |m| m[:dependencies] && enabled_modules.include?(m[:name]) && (m[:dependencies] & enabled_modules) != m[:dependencies] }
```

**Review by aaron-contreras — APPROVED** [review]
⚡

File: app/controllers/admin/settings/projects_settings_controller.rb (hunk 1 above)
Comment (dombesz) [reply app/controllers/admin/settings/projects_settings_controller.rb]: That was fast @aaron-contreras !⚡️  fixed it :)

Comment (oliverguenther) [reply app/controllers/admin/settings/projects_settings_controller.rb]: I didn't even know or couldn't remember the `list_of_messages` helper. Thanks for fixing and reviewing so quickly both of you 🙏

**Pull request merged**